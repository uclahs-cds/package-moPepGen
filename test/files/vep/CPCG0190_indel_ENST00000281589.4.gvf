##fileformat=VCFv4.2
##mopepgen_version=0.0.1
##parser=parseVEP
##reference_index=/data/Index/GRCh38-EBI-GENCODE34
##genome_fasta=
##annotation_gtf=
##source=gindel
##CHROM=<Description='Gene ID'>
##INFO=<ID=TRANSCRIPT_ID,Number=1,Type=String,Description="Transcript ID">
##INFO=<ID=GENE_SYMBOL,Number=1,Type=String,Description="Gene Symbol">
##INFO=<ID=GENOMIC_POSITION,Number=1,Type=String,Description="Genomic Position">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
ENSG00000151846.8	1267	INDEL-1267-AG-A	AG	A	.	.	TRANSCRIPT_ID=ENST00000281589.4;GENOMIC_POSITION=chr13:25097135;GENE_SYMBOL=PABPC3
ENSG00000151846.8	1305	INDEL-1305-TTATGA-T	TTATGA	T	.	.	TRANSCRIPT_ID=ENST00000281589.4;GENOMIC_POSITION=chr13:25097173-25097177;GENE_SYMBOL=PABPC3
ENSG00000151846.8	1327	INDEL-1327-CA-C	CA	C	.	.	TRANSCRIPT_ID=ENST00000281589.4;GENOMIC_POSITION=chr13:25097195;GENE_SYMBOL=PABPC3
