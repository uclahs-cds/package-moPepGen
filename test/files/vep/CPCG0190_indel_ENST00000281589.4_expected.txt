AFSPFGTITSAKVGR
AFSPFGTITSAKVGRWSQQR
FLHLVQSLVQR
FLHLVQSLVQRL
FLHLVQSLVQRLEGGR
FLHLVQSLVQRLEGGRSK
GLVLYVSPPQK
GLVLYVSPPQKKPLK
GLVLYVSPPQKKPLKQLQK
KAFSPFGTITSAKVGR
KPLKQLQK
KRFLHLVQSLVQR
LEGGRSK
LEGGRSKGFGFVCFSSPEEATK
LEGGRSKGLVLYVSPPQK
RFLHLVQSLVQR
RFLHLVQSLVQRL
RFLHLVQSLVQRLEGGR
SKGLVLYVSPPQK
SKGLVLYVSPPQKKPLK
VGRWSQQGFGFVCFSSPEEATK
VGRWSQQR
VGRWSQQRVWFCMFLLPR
VMMEGGRSKGLVLYVSPPQK
VWFCMFLLPR
VWFCMFLLPRR
VWFCMFLLPRRSH
WSQQGFGFVCFSSPEEATK
WSQQRVWFCMFLLPR
WSQQRVWFCMFLLPRR
